lte/gateway/docker/README.md

```diff
@@ -61,14 +61,12 @@ and dirty way to run the containers locally.
 
 ### Running the S1AP integration tests against the containerized AGW
 
-At present, of the S1AP integration tests, it has been verified that `test_attach_detach` can be run versus the containerized AGW.
-To run this test, development mode needs to be enabled when starting the containers, which can be done by using an override
-file `docker-compose.dev.yaml`:
+At present, of the S1AP integration tests, it has been verified that most of the tests can be run versus the containerized AGW;
 
 ```
 cd $MAGMA_ROOT/lte/gateway/docker
 docker-compose down # If containers are already running
-docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up
+docker-compose up
```

Comment:
Doesn't that need to be something like

```suggestion
MAGMA_DEV_MODE=1 docker-compose up
```
or maybe it could be explained in documentation

Reply:
I agree, please modify the line or explain it somewhere

Reply:
The variable is setup with ansible [here](https://github.com/magma/magma/blob/master/orc8r/tools/ansible/roles/gateway_dev/tasks/main.yml). We currently only support testing on the dev vm for the containerized agw.

Reply:
I've tested this locally and indeed you don't need to set it